libmscore/style.h

```diff
@@ -389,6 +388,13 @@ enum class Sid {
     minEmptyMeasures,
     minMMRestWidth,
     mmRestNumberPos,
+    mmRestNumberMaskHBar,
```

Comment:
Most of these should also be properties I believe.

Reply:
I don't think so, see [above](https://github.com/musescore/MuseScore/pull/6211#discussion_r439520555).